app/mixins/custom-form.js

```diff
@@ -286,7 +286,7 @@ export default Mixin.create(MutableArray, {
       this.store.createRecord('custom-form', {
         fieldIdentifier : 'gender',
         form            : 'attendee',
-        type            : 'text',
+        type            : 'select',
```

Comment:
@kushthedude you have to change this in server schema too and the helper functions which create the custom forms automatically.

Reply:
Why make change in schema ? The resulting value is getting stored in, String format .